packages/libs/eda/src/lib/map/analysis/mapTypes/MapTypeHeaderStudyDetails.tsx

```diff
@@ -151,13 +147,20 @@ export function MapTypeHeaderStudyDetails(props: Props) {
           : outputEntity.displayNamePlural}
         {onShowStudies && studyEntity && (
           <>
-            {' '}
-            from {format(studyEntityCount.data.count)}{' '}
-            {studyEntityCount.data.count === 1
-              ? studyEntity.displayName
-              : studyEntity.displayNamePlural}
+            &nbsp;from&nbsp;
```

Comment:
```suggestion
            &nbsp;from
```
My OCD brain can't cope with the extra space before the link. Could we get rid of it please?

Reply:
Without that, there would be no space between "from" and "### Studies"

Reply:
There was for me when I tried it

Reply:
I just tested again. Strange!

Reply:
I'm seeing consistent behavior in all major browsers (on linux) :shrug: